examples/common-voice/scripts/prepare.bash

```diff
@@ -76,10 +78,10 @@ for language in ${datasets[*]}; do
 	total_sec=0
 	total_files=0
 	wavs_dir=$output_dir/$language/16k_wavs
-	mkdir --verbose $wavs_dir
+	mkdir -pv $wavs_dir
```

Comment:
```suggestion
	mkdir --verbose $wavs_dir
```

Reply:
The extra `-p` flag on line 81 is to stop it from throwing an error if the directory already exists - which can happen if running the script twice due to an error earlier (amongst other reasons)